src/pages/create/TypeStep.tsx

```diff
@@ -1,15 +1,41 @@
 import { Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
 
+import { OfferType } from '@/constants/OfferType';
 import type { StepProps } from '@/pages/Steps';
-import { Text } from '@/ui/Text';
+import { Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
+import { ToggleBox } from '@/ui/ToggleBox';
 
 const TypeStep = ({ control, field }: StepProps<FormData>) => {
+  const { t } = useTranslation();
+
   return (
     <Controller
       control={control}
       name={field}
       render={({ field }) => {
-        return <Text>{field.name}</Text>;
+        return (
+          <Inline
+            spacing={5}
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <ToggleBox
+              onClick={() => field.onChange(OfferType.EVENT)}
+              active={field.value === OfferType.EVENT}
+              icon={Icons.CALENDAR_ALT}
+              text={t('offerTypes.event')}
+            />
+            <span css="font-style: italic;">{t('steps.offerTypeStep.or')}</span>
```

Comment:
I would use a Text component here, if we don't have a good variant we can add some css on the element itself